.travis.yml

```diff
@@ -66,23 +66,22 @@ jobs:
       - git branch -f "$TRAVIS_BRANCH" && git checkout "$TRAVIS_BRANCH" && sbt 'set credentials += Credentials("whitesource", "whitesourcesoftware.com", "", System.getenv("WHITESOURCE_KEY"))' whitesourceCheckPolicies whitesourceUpdate
     name: "Check licenses with WhiteSource"
 
+  - stage: publish
+    script: sbt -J-XX:ReservedCodeCacheSize=128m +publish
+    name: Publish artifacts
+  - script: eval "$(ssh-agent -s)" && cp .travis/id_rsa /tmp/id_rsa && chmod 600 /tmp/id_rsa && ssh-keygen -p -P "$DEPLOY_PASSPHRASE" -N "" -f /tmp/id_rsa && ssh-add /tmp/id_rsa && sbt docs/publishRsync
+    name: Publish documentation
+
 stages:
   - name: check
     if: NOT tag =~ ^v
   - name: test
     if: NOT tag =~ ^v
   - name: whitesource
     if: repo = akka/akka-management AND ( ( branch = master AND type = push ) OR tag =~ ^v )
+  - name: publish
+    if: repo = akka/akka-management AND ( ( branch = master AND type = push ) OR tag =~ ^v
```

Comment:
so this also introduces artifact snapshot publishing? where do those end up?

Reply:
Ahh. Thats an oversights. Before the changes, `deploy` section would only run on tagged commits.